### PR TITLE
Homework #16

### PR DESCRIPTION
**Pull request by ElizaTrip**
- Added drf-yasg.
- - Added django-cors-headers.

### CODE REVIEW

File: .github/workflows/app.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ name: Python application
 
 on:
   push:
-    branches: [ master, rest ]
+    branches: [ master, swagger ]
```
Comment (valeriiduz) [inline .github/workflows/app.yml:8]: Its not important, you can not change this rule, better to put only master in your case

File: .github/workflows/app.yml [hunk 2]
```diff
@@ -31,6 +31,8 @@ jobs:
         pip install flake8 pytest
         pip install python-dotenv
         pip install djangorestframework
+        pip install -U drf-yasg
```
Comment (valeriiduz) [inline .github/workflows/app.yml:34]: Use requerments.txt file instead install each package separate

File: src/app/views.py [hunk 3]
```diff
@@ -153,18 +153,27 @@ def activate(request, uid, token):
 
 
 class PersonViewSet(viewsets.ModelViewSet):
+    """
+        API endpoint that allows person to be viewed or edited.
```
Comment (valeriiduz) [inline src/app/views.py:157]: `API endpoint that allows` duplicate in each your ViewSet. Better write something like that
`Person ViewSet which allow list, create, retrieve, update and delete methods`

File: src/app/views.py [hunk 4]
```diff
@@ -153,18 +153,27 @@ def activate(request, uid, token):
...
+    """
     queryset = Person.objects.all()
     serializer_class = PersonSerializer
     permission_classes = [permissions.IsAuthenticated]
 
 
 class GroupViewSet(viewsets.ModelViewSet):
+    """
+         API endpoint that allows Group to be viewed or edited.
```
Comment (valeriiduz) [inline src/app/views.py:166]: the same as above

File: src/app/views.py [hunk 5]
```diff
@@ -153,18 +153,27 @@ def activate(request, uid, token):
...
+    """
     queryset = Group.objects.all()
     serializer_class = GroupSerializer
     permission_classes = [permissions.IsAuthenticated]
 
 
 class SubjectViewSet(viewsets.ModelViewSet):
+    """
+     API endpoint that allows Subjects to be viewed or edited.
```
Comment (valeriiduz) [inline src/app/views.py:175]: the same as above

File: src/settings/settings.py [hunk 6]
```diff
@@ -25,15 +25,15 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 load_dotenv(dotenv_path=BASE_DIR / '../.env')
 
-SECRET_KEY = os.getenv('SECRET_KEY')
+SECRET_KEY = 'django-insecure-=srfh1oo-o@75*n1d)0b(3_q4!qd(x%i8%gp70gxi3-rgfda(#'
```
Comment (valeriiduz) [inline src/settings/settings.py:28]: use .env file

Comment (ElizaTrip) [reply src/settings/settings.py:28]: Secret key used to be in .env file. But yesterday's job ended with an error that occurred while checking templates.

image

So I temporarily moved secret key to settings. I wanted to ask you how to fix this error, but I didn't have time yesterday. Should I just remove template validation from my app.yml file or is there another way?

**Pull request merged**